keras/src/backend/mlx/numpy.py

```diff
@@ -158,13 +158,13 @@ def arctanh(x):
     return mx.arctanh(x)
 
 
-def argmax(x, axis=None):
+def argmax(x, axis=None, keepdims=False):
     x = convert_to_tensor(x)
-    return mx.argmax(x, axis=axis)
+    return mx.argmax(x, axis=axis, keepdims=keepdims)
 
 
-def argmin(x, axis=None):
-    return mx.argmin(x, axis=axis)
+def argmin(x, axis=None, keepdims=False):
+    return mx.argmin(x, axis=axis, keepdims=keepdims)
```

Comment:
do we need to add convert_to_tensor to all funcs? Few have it and others don't.

Reply:
Generally, yes, we should always use it on tensor arguments.